config/environments/production.rb

```diff
@@ -1,10 +1,12 @@
 Rails.application.configure do
   config.action_mailer.smtp_settings = {
-    address: ENV['SMTP_HOSTNAME'],
-    port: ENV['SMTP_PORT'],
-    domain: ENV['SMTP_DOMAIN'],
-    authentication: :login,
-    enable_starttls_auto: true
+    user_name:             ENV['SMTP_USERNAME'],
+    password:              ENV['SMTP_PASSWORD'],
+    address:               ENV['SMTP_HOSTNAME'],
+    port:                  ENV['SMTP_PORT'],
+    domain:                ENV['SMTP_DOMAIN'],
+    authentication:        :login,
+    enable_starttls_auto:  true
```

Comment:
I'd prefer not to line things up: it ends up making commits noisier and merges harder than they'd otherwise be.